data/blood/glucose/glucose.go

```diff
@@ -60,3 +61,49 @@ func NormalizeValueForUnits(value *float64, units *string) *float64 {
 	}
 	return value
 }
+
+const (
+	// MgdL_Per_MmolL renames MmolLToMgdLConversionFactor in an alternate way.
+	MgdL_Per_MmolL = MmolLToMgdLConversionFactor
```

Comment:
I know, underscores make it more readable, but we lose any consistency we have with the rest of the code. Something to consider.

Reply:
Also, why a new name?

Reply:
> I know, underscores make it more readable, but we lose any consistency we have with the rest of the code. Something to consider.

Yes, I generally hate underscores in Go names. This was an intentional choice for readability. As I see it, since this is important code, readability trumps convention. For that matter, I personally _don't_ find that `AThingWithALongName` is less readable than `A_Thing_With_A_Long_Name`, but in this specific case, with the MgdL and MmolL, I found the underscores clearer.

> Also, why a new name?

I thought it was more readable. It took me a while, and several readings of the code, to figure out how to use `MmolLToMgdLConversionFactor` correctly. Assuming I have MmolL, do I multiply or divide to get MgdL? It's not clear to me from the name. 

With MgdL_Per_MmolL, from the name alone I know that this value is a ratio of:
```
  Mg/dL
  -----
  Mmol/L
```
So then, if I have MmolL, and want to get MgdL, I know that I can multiply my value by MgdL_Per_MmolL and the MmolL will cancel out, leaving me with MgdL.

This could all be just a result of how my brain works, or how I learned to do these conversion in school or something, I dunno. If you don't find the new names more readable, I can leave them out.

Reply:
Well, I'll leave that up to you to decide. However, consider this from a perspective of a future engineer looking at this code and seeing two constants referring to the same value. Future me would ask "why two constants here"?

I agree that `MmolLToMgdLConversionFactor` is poorly named. I told you I suck at naming things. ;)

Fortunately, it looks like `MmolLToMgdLConversionFactor` (and `MmolLToMgdLPrecisionFactor`) is only used in `glucose.go` and `glucose_test.go`, so consider just renaming it to something more descriptive. This will directly address the root of the problem (a poorly named constant). :100:

Reply:
I'm not a fan of having two names for the same thing at all, but I was unwilling to touch _anything_ in the other functions, given the sensitivity of the method. In hindsight, that was silly, and it's better to make those changes (if I think they're warranted) and then call out dangerous areas to code reviewers for careful review.

If you're not opposed to `MmolLToMgdLConversionFactor` being renamed to `MgdL_Per_MmolL`, I would gladly make that switch and collapse them back down to 1 constant.

Naming is hard, I won't deny it, or pretend that I'm somehow better than anyone else at it. I can only explain my reasoning and hope it resonates with future readers.